source/dbo/Stored Procedures/Unmatched Specimens/uspGenerateUnmatchedSpecimensLinkedNotifications.sql

```diff
@@ -23,42 +23,45 @@ BEGIN TRY
 	--Exclude any matches that have previously been rejected (otherwise every rejection has an accompanying match)
 	LEFT JOIN #PreviousMatch pm on pm.NotificationID = mr.rclientsourceID AND pm.ReferenceLaboratoryNumber = mr.clientsourceID
 	LEFT JOIN #PreviousPossibleMatch ppm on ppm.NotificationID = mr.rclientsourceID AND ppm.ReferenceLaboratoryNumber = mr.clientsourceID
+	JOIN [$(NTBS_Specimen_Matching)].dbo.MatchingConfiguration mc ON mr.[weight] < mc.AutoMatch
```

Comment:
Doing this as a `JOIN` is semantically strange. I think this code will be easier to follow if you select the value into a variable, and then use a `WHERE` clause as you were doing before.